app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
No, not now. I want to say, build checks data binding files too. Remember the `checkedIn()` issue when refactoring to nextgen..

Just to confirm the file names added here will be excluded for syntax check right? If yes then we should include data binding generated files too. As we don't edit them.